CHANGELOG.md

```diff
@@ -28,6 +28,14 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
     
+## dev - 2021-02-16
```

Comment:
@NLaws since this is going into master, at this point we can tag the release number. I think this make sense because if we wait till after the PR goes through to update the number here develop and master will show up as being out-of-sync